src/test/java/seedu/address/logic/commands/EditPetDescriptorTest.java

```diff
@@ -44,7 +44,7 @@ public void equals() {
         assertFalse(DESC_AMY.equals(editedAmy));
 
         // different email -> returns false
-        editedAmy = new EditPetDescriptorBuilder(DESC_AMY).withOwnerName(VALID_OWNERNAME_BOB).build();
+        editedAmy = new EditPetDescriptorBuilder(DESC_AMY).withOwnerName(VALID_OWNER_NAME_BOB).build();
```

Comment:
Same as the comments above; should the `EditPetDescriptorBuilder` be named after the pet's name instead?

Reply:
Similar situation as described above.